### ISSUE TITLE
Fuel list key isn't dynamic

### ISSUE THREAD

**Issue by chrispikula** [CleverRaven/Cataclysm-DDA#59698]
### Describe the bug

![image](https://user-images.githubusercontent.com/22281606/181458130-01ebf008-917f-436b-8857-582becb57636.png)
Uses 
[ and ] to scroll, not
> and <
as per
![image](https://user-images.githubusercontent.com/22281606/181458227-c471e958-5f94-4fcd-b1f7-1fd38f412f3a.png)

### Steps to reproduce

Have a lot of stuff

### Expected behavior

That > and < would be ] and [

### Versions and configuration

Win 10 Tiles, 42a7fde

**Comment by hexagonrecursion** [comment]
Do you mean that > in this image should be replaced by whichever key is currently assigned?
![image](https://user-images.githubusercontent.com/52621858/181680838-e5fc151b-f3cc-46e4-8baf-2ac9ee64d9a8.png)


- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 36 (Thirty Six); Release: 36; Codename: ThirtySix; 
- Game Version: 4abdbc8 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by chrispikula** [comment]
Yes, that is what I implied.  I seem to remember that there was shown option of the keys used to scroll there, back mid 2021, but now it's not.  Add to that that > and < are frequently used to scroll through other menus, and it implies that that's the key to use.
